test/common/router/config_impl_test.cc

```diff
@@ -6972,7 +6936,6 @@ name: RetriableStatusCodes
 
 TEST_F(RouteConfigurationV2, BadConnectConfig) {
   const std::string yaml = R"EOF(
-name: RetriableStatusCodes
```

Comment:
Example of the name property not matching the test name because it was copy/pasted from another test.